modules/op-results-storage.adoc

```diff
@@ -60,6 +82,30 @@ $ oc create secret generic gcs-credentials \
 ----
 +
 Adjust the path and filename of the application credentials file as necessary.
++
+.. Edit the `TektonConfig` custom resource (CR) by using the following command:
++
+[source,terminal]
+----
+$ oc edit TektonConfig config
+----
++
+Make the following changes in the `result` spec:
++
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  result:
+    gcs_creds_secret_name: gcs-credentials
```

Comment:
This isn't supported. It might work but we don't support this.

Reply:
So what is the correct way to use Google Cloud Storage? Or should I remove the Google Cloud Storage option entirely?

Reply:
Also, shall I keep the other two storage options, S3 Bucket and PVC? Or is it just LokiStack now? @khrm